app/assets/javascripts/app/results-file-upload.js

```diff
@@ -0,0 +1,16 @@
+function makeAModal() {
+  new AjaxModal('a.results-file-upload', '#results-file-upload-modal', {
```

Comment:
This class and id are only used in this order show table.  I think we should move this code into `app/assets/javascripts/order_table_refresh.js`

Reply:
Also let's use " instead of '